src/tests/packages/utils.unmocked.test.ts

```diff
@@ -0,0 +1,63 @@
+import { decodePngMetadata, decodeSvgMetadata } from "../../data/image";
+import { ImportedDataState } from "../../data/types";
+import * as utils from "../../packages/utils";
+import { API } from "../helpers/api";
+
+describe("embedding scene data", () => {
+  describe("exportToSvg", () => {
+    it("embedding scene data shouldn't modify them", async () => {
```

Comment:
```suggestion
    it("shouldn't modify the scene", async () => {
```
If you want to keep the describe block, lets rename this to be more clear and avoid redundancy